crypto/ts/types.ts

```diff
@@ -1,10 +1,24 @@
-export type SnarkBigInt = bigint;
-export type PrivKey = bigint;
-export type PubKey = bigint[];
-export type Point = bigint[];
-export type EcdhSharedKey = bigint[];
-export type Plaintext = bigint[];
-export type Ciphertext = bigint[];
+// we define a bignumber as either a bigint or a string
+// which is what we use the most in MACI
+export type BigNumber = bigint | string;
```

Comment:
I think this type can have conflicts if it's using with BigNumber from ethers of from bignumber.js. Maybe it makes sense to reuse BigNumberish type from ethers?

Reply:
Good shout, I can use that instead

Reply:
What do you think about simply changing the type name? so we don't import from ethers and make further changes to the code

Reply:
I think it works as well.